libs/minisat/Vec.h

```diff
@@ -94,6 +94,8 @@ class vec {
 };
 
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wclass-memaccess"
```

Comment:
This needs some kind of ifdef guard. Otherwise for example with GCC 7.4.0:

```
test.cc:3:32: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
 #pragma GCC diagnostic ignored "-Wclass-memaccess"
                                ^~~~~~~~~~~~~~~~~~~
```

or with clang 6.0.0:

```
test.cc:3:32: warning: unknown warning group '-Wclass-memaccess', ignored [-Wunknown-warning-option]
#pragma GCC diagnostic ignored "-Wclass-memaccess"
```

Reply:
Oh, that's very ugly. Maybe it's better to just have the warning, and then someone can think hard about writing this realloc stuff in a way that is more pleasing both for humans and gcc-9?

For now, I'll just remove the pragmas, then?